app.rb

```diff
@@ -70,6 +70,22 @@ def set_locale(locale)
     redirect "/"
   end
 
+  put '/rewind' do
+    game = session[:game]
+    if game.board.unplayed?(game.current_state)
```

Comment:
why does this need to pass in a `game.current_state`? does the game object already have access to `current_state`?